scripts/pushArgos.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+const util = require('util');
+const glob = require('fast-glob');
+const fse = require('fs-extra');
+const chunk = require('lodash/chunk');
+const childProcess = require('child_process');
+
+const execFileAsync = util.promisify(childProcess.execFile);
+
+async function exec(command, args) {
```

Comment:
```suggestion
const execFileNode = util.promisify(childProcess.execFile);

async function execFile(command, args) {
```

To avoid confusion with node's `exec` which has a different signature.